site/app/layout.tsx

```diff
@@ -31,9 +25,6 @@ export default function RootLayout({
           <link rel="icon" href="/icon.svg" type="image/svg+xml" />
         </head>
         <body className="flex flex-col min-h-screen font-body antialiased bg-putty selection:bg-pink-neon/60">
-          <Suspense>
-            <PostHogPageView />
-          </Suspense>
```

Comment:
figured this made more sense in the analytics provider, so I moved it over there while I was at it. idk if this is going against some prior reasoning.